contrib/cliphist-fuzzel-img

```diff
@@ -1,27 +1,24 @@
 #!/usr/bin/env bash
 
-# requires imagemagick to generate thumbnails
-
-thumbnail_size=64
 thumbnail_dir="${XDG_CACHE_HOME:-$HOME/.cache}/cliphist/thumbnails"
 
 cliphist_list=$(cliphist list)
 if [ -z "$cliphist_list" ]; then
-  fuzzel -d --placeholder "cliphist: please store something first" --lines 0
+  fuzzel -d --prompt-only "cliphist: please store something first "
```

Comment:
Changed from placeholder to prompt because of conflicts with `minimal-lines`. This could be changed to `fuzzel -d --prompt-only "" --placeholder "cliphist: please store something first "` to keep placeholder style.